projects/plugins/boost/phpunit.xml.dist

```diff
@@ -17,6 +17,10 @@
 
 	<filter>
 		<whitelist processUncoveredFilesFromWhitelist="false">
```

Comment:
As best I can tell, the above is [the same as the default](https://github.com/sebastianbergmann/php-code-coverage/blob/34cc484af1ca149188d0d9e91412191e398e0b67/src/CodeCoverage.php#L56-L64), right? Or am I missing some nuance as to why we need this attribute?

```suggestion
		<whitelist>
```

Reply:
Yeah, all these could use updating. For now I just copied this from `tools/cli/skeletons/`